Source/PINDiskCache.m

```diff
@@ -1025,7 +1094,9 @@ - (void)setObject:(id <NSCoding>)object forKey:(NSString *)key fileURL:(NSURL **
             if (date) {
                 self->_metadata[key].date = date;
             }
-            
+            if (ageLimit > 0.0) {
+                [self asynchronouslySetAgeLimit:ageLimit forURL:fileURL];
```

Comment:
Not sure the answer to this: will there be implications to this being asynchronous? Specifically that _metadata[key].ageLimit won't be set immediately and checks against it could happen before it is?

I'm *guessing* the answer is no, but maybe there could be a comment here suggesting it's been thought through and why it's safe?